EasyPost.Tests/Fixtures.cs

```diff
@@ -0,0 +1,144 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using EasyPost.Tests.FixtureData;
+using EasyPost.Utilities;
+
+namespace EasyPost.Tests
+{
+    public static class Fixtures
+    {
+        public static byte[] EventBody
+        {
+            get
+            {
+                // need to go up one level to get out of the EasyPost.Tests directory
+                const string relativePath = "../examples/official/fixtures/event-body.json";
+                string fullPath = Path.Combine(TestUtils.GetSourceFileDirectory(), relativePath);
+
+                try
+                {
+                    string jsonString = File.ReadLines(fullPath).First();
+                    return Encoding.UTF8.GetBytes(jsonString);
+                }
+                catch (Exception e)
+                {
+                    throw new Exception($"Unable to read {fullPath}", e);
+                }
+            }
+        }
+
+        internal static Dictionary<string, object> BasicCarrierAccount => GetFixtureStructure().CarrierAccounts!.Basic!;
+
+        internal static Dictionary<string, object> BasicCustomsInfo => GetFixtureStructure().CustomsInfos!.Basic!;
+
+        internal static Dictionary<string, object> BasicCustomsItem => GetFixtureStructure().CustomsItems!.Basic!;
+
+        internal static Dictionary<string, object> BasicInsurance => GetFixtureStructure().Insurances!.Basic!;
+
+        internal static Dictionary<string, object> BasicOrder => GetFixtureStructure().Orders!.Basic!;
+
+        internal static Dictionary<string, object> BasicParcel => GetFixtureStructure().Parcels!.Basic!;
+
+        internal static Dictionary<string, object> BasicPickup
+        {
+            get
+            {
+                Dictionary<string, object> fixture = GetFixtureStructure().Pickups!.Basic!;
+
+                string pickupDate = "2022-08-30";
```

Comment:
nit: add a comment here `If you need to re-record cassettes, increment the date below and ensure it is one day in the future,`

Reply:
Resolved.